osgp/protocol-adapter-dlms/osgp-dlms/src/main/java/org/opensmartgridplatform/dlms/services/ObjectConfigService.java

```diff
@@ -313,6 +313,22 @@ public static List<String> getCaptureObjectDefinitions(final CosemObject profile
     return List.of(captureObjectDefinitions);
   }
 
+  public static List<String> getSourceObjectDefinitions(final CosemObject profile) {
```

Comment:
ik zou hier een specifieke unit test voor verwachten.

Reply:
This method is tested in the  DlmsProfileValidatorTest

Reply:
Okay, you are right, I will add test for this method in ObjectConfigServiceTest